README.md

```diff
@@ -7,65 +7,75 @@ SweFreq - Swedish Frequency database
 Running on a production system
 ------------------------------
 
-The application has only been tested with python 3.5.2. It will most likely work with at least other 3.5 versions.
+The application is targeted at python 3.6. It will most likely work with later versions as well.
 
-`virtualenv` is not a requirement but it will help you to install the application.
+`venv` is not a requirement but it will help you to install the application.
 
 1. Download the repository from [github](https://github.com/NBISweden/swefreq)
 
-2. Credentials
-
-   2.1 Make sure you have registered an Elixir AAI application with the Elixir.
-
-   2.2 (legacy) Create Google OAuth 2.0 credentials with the [Google API
-   Console](https://console.developers.google.com/) for enabling
-   authentication.
+2. Make sure you have registered an Elixir AAI application with the Elixir.
 
 3. Rename the file `settings_sample.json` into `settings.json` and edit all
    the values.
 
-4. Install MySQL. Only tested with 5.6. Note, the schema is incompatible with the
-   latest version of MariaDB.
+5. If you have `venv` and `pip` installed then you can do the following
```

Comment:
I know that numbers don't matter for _rendering_ markdown, it's still nice if they are consequtive in the file.

Reply:
Oops. Kept on checking how it looked rendered, but never saw that.